### PR TITLE
Add patches

### PR DESCRIPTION
**Pull request by owenherbert**
I've added the latest changes from our forked repository.

### CODE REVIEW

File: lang/en/block_progress.php [hunk 1]
```diff
@@ -251,5 +253,3 @@
 // For My home page.
 $string['no_blocks'] = "No Progress Bar blocks are set up for your courses.";
 $string['no_courses'] = "You are not enrolled in any courses. Only bars from enrolled courses will be shown.";
-
-$string['privacy:metadata'] = "Progress block does not store any personal data.";
```
Comment (golenkovm) [inline lang/en/block_progress.php]: I'm not sure why we delete this

File: version.php [hunk 2]
```diff
@@ -24,8 +24,8 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-$plugin->version   = 2016081800;
+$plugin->version   = 2016081800.01;
```
Comment (golenkovm) [inline version.php]: I would also update to `2016081801` for this PR

File: lang/en/block_progress.php (hunk 1 above)
Comment (owenherbert) [reply lang/en/block_progress.php]: I've added this item back in, it must have slipped while resolving conflicts: https://github.com/catalyst/moodle-block_progress/pull/1/commits/6eff21e979c9cc892ef99ed1944d9e297c1769b3 .

File: version.php [hunk 3]
```diff
@@ -24,8 +24,8 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-$plugin->version   = 2016081800;
+$plugin->version   = 2016081801.01;
```
Comment (golenkovm) [inline version.php]: Ah, sorry @owenherbert  I meant to make it `2016081801` without decimal numbers

**Comment by golenkovm** [comment]
Thanks @owenherbert LGTM :+1:

**Pull request merged**